### ISSUE TITLE
Beta - 'Random Numbers' typos in instructions

### ISSUE THREAD

**Issue by wildlifehexagon** [FreeCodeCamp/freecodecamp#1590]
Challenge http://beta.freecodecamp.com/challenges/random-numbers has an issue. Please describe how to reproduce it, and include links to screenshots if possible.

This isn't terribly important, but I noticed a few grammar issues in the instructions for this challenge. Maybe reword it as such?

---

Random numbers are a very useful way to create random behaviors and games.

JavaScript has a Math.random() method that can generate a random decimal number.

Let's have a go of Math.random() now by getting myFunction to return a random number.

**Comment by Greenheart** [comment]
I'll take care of this one! Will be some good practice with the git workflow :)

**Comment by Greenheart** [comment]
Pull request submitted, #1607

**Comment by Greenheart** [comment]
@benmcmahon100 let's close this one

**Comment by benmcmahon100** [comment]
@Greenheart when making a pr do have a line in the commit message like this
```
Fix (The url of the issue)
```
That way once the pr is merged it will auto-close the issue (I think)

**Comment by Greenheart** [comment]
@benmcmahon100 Thanks! I'll do that in the future! :)

**Comment by benmcmahon100** [comment]
@Greenheart Awesome keep up the good work!
